sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -230,7 +231,7 @@ def create_database(  # pylint: disable=redefined-builtin
         self,
         id,  # type: str
         populate_query_metrics=None,  # type: Optional[bool]
-        offer_throughput=None,  # type: Optional[int]
+        offer_throughput=None,  # type: Optional[Union[int, ThroughputProperties]]
```

Comment:
Let's add this to the docstring as well - though it will probably need to be two lines to get the right formatting (this applies to all the docstring updates - whether 'keyword' or 'param'):
```python
"""
:param offer_throughput: The provisioned throughput for this offer.
:paramtype offer_throughput: int or ~azure.cosmos.ThroughputProperties
"""
```